sdk/storage/storage-blob/samples/typescript/iterators.ts

```diff
@@ -0,0 +1,89 @@
+/* 
+ Setup: Enter your storage account name and shared key in main()
+*/
+
+import {
+  ContainerClient,
+  BlobServiceClient,
+  StorageClient,
+  SharedKeyCredential,
+  BlobClient,
+  BlockBlobClient
+} from "../../src"; // Change to "@azure/storage-blob" in your package
+
+async function main() {
+  // Enter your storage account name and shared key
+  const account = process.env["ACCOUNT_NAME"] || "";
+  const accountKey = process.env["ACCOUNT_KEY"] || "";
```

Comment:
nit: be consistent with other sample, so no `process.env["ACCOUNT_NAME"] || ` and `process.env["ACCOUNT_KEY"] || `

Reply:
or Should we consider adding this in the other samples as well?

Reply:
I forgot where but I seem to remember some advice on not asking users to put sensitive data in env variables.

Reply:
Oh, makes sense, any application would be able to access the info.
Will remove this in the sample.

Then, the way we use `dotenv` is also not safe 🤔 ?

Reply:
> Then, the way we use dotenv is also not safe 🤔 ?

it may be fine for development?

Reply:
Hmmm.